### PR TITLE
Support language switching in wordpad

### PR DESCRIPTION
**Pull request by feerrenrut**
Fix for #6555

One concern is to ensure that we dont re-introduce #2047: don't expose language for whitespace as its incorrect for east-asian languages. I'm not sure how to test for that specifically.

### CODE REVIEW

**Comment by feerrenrut** [comment]
@michaelDCurran Would you mind reviewing this?

File: source/NVDAObjects/window/edit.py [hunk 1]
```diff
@@ -532,6 +541,16 @@ def _getFormatFieldAtRange(self,range,formatConfig):
 				formatField['background-color']=_("Unknown color")
 			else:
 				formatField["background-color"]=colors.RGB.fromCOLORREF(bkColor)
+		if True:
```
Comment (michaelDCurran) [inline source/NVDAObjects/window/edit.py]: old debugging code?

**Comment by michaelDCurran** [comment]
Perhaps move Winword's _getLanguageFromLCID into languageHandler, and use it also in edit.py rather than duplicating it.
Other than that, this looks good.

**Comment by michaelDCurran** [comment]
Looking good to me. Re #2047, from the quick testing I have done, it looks like Wordpad does not suffer from the same bug in MS word. However,  if a user of east-asian languages does end up finding a testcase showing up the bug, we have code that can fix it easily enough.

**Comment by feerrenrut** [comment]
Ok, in that case I will go ahead and incubate this.

**Comment by feerrenrut** [comment]
This PR is replaced by #6626

**Pull request closed without merging**